client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -452,6 +454,12 @@ export default function WPCheckout( {
 			</CheckoutSummaryArea>
 			<CheckoutStepArea
 				submitButtonHeader={ <SubmitButtonHeader /> }
+				submitButtonFooter={
+					<SubmitButtonFooter>
+						<img src={ badge14Src } alt="" />
+						<span>{ translate( '14 day money back guarantee' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( '7 day money back guarantee' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/improve/checkout-money-back-mention)